packages/design-picker/src/constants.ts

```diff
@@ -45,6 +46,13 @@ export const FEATURE_CATEGORIES = {
 	STORE: 'store',
 };
 
+export const DESIGN_TIER_CATEGORIES = {
+	FREE: {
+		slug: FREE_THEME,
+		name: translate( 'Free' ),
```

Comment:
The translation may not work before the locale is initialized.

Reply:
Thanks @arthur791004 for taking care of this.